src/test/ly/stealth/mesos/kafka/ExprTest.scala

```diff
@@ -76,6 +78,31 @@ class ExprTest extends MesosTestCase {
   }
 
   @Test
+  def expandBrokers_sortByAttrs {
+    val cluster = Scheduler.cluster
+    val b0 = cluster.addBroker(new Broker("0"))
+    val b1 = cluster.addBroker(new Broker("1"))
+    val b2 = cluster.addBroker(new Broker("2"))
+    val b3 = cluster.addBroker(new Broker("3"))
+    val b4 = cluster.addBroker(new Broker("4"))
+    val b5 = cluster.addBroker(new Broker("5"))
+
+    b0.task = new Broker.Task(_attributes = Util.parseMap("r=2,a=1"))
+    b1.task = new Broker.Task(_attributes = Util.parseMap("r=0,a=1"))
+    b2.task = new Broker.Task(_attributes = Util.parseMap("r=1,a=1"))
+    b3.task = new Broker.Task(_attributes = Util.parseMap("r=1,a=2"))
+    b4.task = new Broker.Task(_attributes = Util.parseMap("r=0,a=2"))
+    b5.task = new Broker.Task(_attributes = Util.parseMap("r=0,a=2"))
+
+    assertEquals(util.Arrays.asList("0", "1", "2", "3", "4", "5"), Expr.expandBrokers(cluster, "*", sortByAttrs = true))
+    assertEquals(util.Arrays.asList("1", "2", "0", "4", "3", "5"), Expr.expandBrokers(cluster, "*[r]", sortByAttrs = true))
+    assertEquals(util.Arrays.asList("1", "4", "2", "3", "0", "5"), Expr.expandBrokers(cluster, "*[r,a]", sortByAttrs = true))
```

Comment:
this is great too, attributes being set based on the disk configuration becomes useful here too, different topics often have different disk configurations so this can pin both cases same time, great